test/cases/parsing/nullish-coalescing/index.js

```diff
@@ -0,0 +1,20 @@
+it("should parse nullish coalescing correctly", () => {
+	let result;
+
+	if ((null ?? false) === null) {
+		result = require("./b");
+	} else if ((0 ?? false) === 0) {
+		result = require("./a");
+	}
+
+	expect(result).toBe(1);
+});
+
+it("should evaluate module.hot to nullish", () => {
+	if (module.hot ?? true) {
```

Comment:
Should I move this to `configCases` since this does not work with `TestCasesHot`?